python/pyspark/sql/tests.py

```diff
@@ -2504,6 +2504,10 @@ def test_conf(self):
         spark.conf.unset("bogo")
         self.assertEqual(spark.conf.get("bogo", "colombia"), "colombia")
 
+        self.assertRaisesRegexp(Exception, "hyukjin", lambda: spark.conf.get("hyukjin"))
```

Comment:
@BryanCutler, so this test case prompt you. It's not related with this PR:

**Before**

```
>>> spark.conf.get("hyukjin")
...
: java.util.NoSuchElementException: hyukjin
...
```

**After**


```
>>> spark.conf.get("hyukjin")
...
: java.util.NoSuchElementException: hyukjin
...
```

Let me take this out.